config/templates/base.html

```diff
@@ -36,6 +36,11 @@
     </head>
 
     <body class="{% block body_class %}{% endblock %}">
+        {% block navbar %}
+            {% include "navbar.html" %}
+        {% endblock %}
+
+        {# Wagtail userbar #}
```

Comment:
Hi @Kands221 this puts the navbar on top of the existing promotional banner. Can I bother you to move this block: https://github.com/pythonph/pyconapac-2025/blob/61197a506e0b4348801297b1cd395a9a0078df99/pyconph/home/templates/home/home_page.html#L8 from home_page to base.html and put it on top of navbar component?

Same with what's in `templates/content/content_page.html`